lib/index.js

```diff
@@ -18,7 +18,7 @@ const schema = Joi.object({
 
```

Comment:
In the schema above, I believe `Joi.func()` was renamed to `Joi.function()`. Though I suppose there may still be an alias for it...